src/models/m_rsc_export.erl

```diff
@@ -142,10 +142,12 @@ privacy_filter(Export) ->
         fun(P, Acc) ->
             proplists:delete(P, Acc)
         end,
-        Drop,
-        Export).
+        Export,
+        Drop
+    ).
 
-%% @doc Given an edge record, add the resource uris for the object and the predicate.
+%% @doc Given an edge record, add the resource uris for the object and the
+%% predicate.
 edge_details(Edge, Context) ->
     Edge ++ [{predicate_uri, m_rsc:p(proplists:get_value(predicate_id, Edge), uri, Context)},
              {predicate_title, m_rsc:p(proplists:get_value(predicate_id, Edge), title, Context)},
```

Comment:
Line 153 is too long:              {predicate_title, m_rsc:p(proplists:get_value(predicate_id, Edge), title, Context)},.